nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/java/org/nuxeo/ecm/platform/audit/service/LogEntryProvider.java

```diff
@@ -106,6 +106,18 @@ public void addLogEntries(List<LogEntry> entries) {
     }
 
     @SuppressWarnings("unchecked")
+    public List<LogEntry> getLogEntriesFor(String uuid, String repositoryId) {
+        if (log.isDebugEnabled()) {
+            log.debug("getLogEntriesFor() UUID=" + uuid + " and repositoryId=" + repositoryId);
+        }
+        Query query = em.createNamedQuery("LogEntry.findByDocumentAndRepository");
+        query.setParameter("docUUID", uuid);
+        query.setParameter("repositoryId", repositoryId);
+        return doPublish(query.getResultList());
+    }
+
+    @SuppressWarnings("unchecked")
+    @Deprecated
```

Comment:
@efge there is no interface signature for this method

@egiuly-nuxeo there are deprecated usages to fix here too